runtime/src/lib.rs

```diff
@@ -262,15 +264,41 @@ impl pallet_transaction_payment::Config for Runtime {
 	type FeeMultiplierUpdate = ();
 }
 
+// parameter_types! {
+// 	pub const TombstoneDeposit: Balance = 16 * MILLICENTS;
+// 	pub const RentByteFee: Balance = 100_000;
+// 	pub const RentDepositOffset: Balance = 1000 * MILLICENTS;
+// 	pub const SurchargeReward: Balance = 150 * MILLICENTS;
+// 	pub const SignedClaimHandicap: u32 = 2;
+// 	pub const MaxDepth: u32 = 32;
+// 	pub const StorageSizeOffset: u32 = 8;
+// 	pub const MaxValueSize: u32 = 16 * 1024;
+// }
```

Comment:
These are the existing contracts parameters, I have for now copied over the new parameters from the `node-runtime` :point_down: , so these probably need updating with some help from @athei

Reply:
Using values copied from substrate node runtime